lib/bundle.js

```diff
@@ -164,7 +165,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     if (opts.injectConfig)
       ui.log('ok', '`' + output.bundleName + '` added to config bundles.');
 
-    logBuild(path.relative(process.cwd(), fileName), opts);
+    logBuild(path.relative(env.cwd(), fileName), opts);
```

Comment:
It would be nice to do away with the `env` as a separate file and mode here and just work with `process.cwd()` here. Note process.cwd() can be set via https://nodejs.org/api/process.html#process_process_chdir_directory if necessary in the jspm.js bootstrap.